recipe/conda_build_config.yaml

```diff
@@ -200,12 +206,12 @@ vc:
   - 14
   - 14
 vlfeat:
-  - 0.9
-xerces_c:
-  - 3
+  - 0.9.20
+xerces-c:
+  - 3.2.0
 xz:
-  - 5
+  - 5.2
 zeromq:
   - 4.2
 zlib:
-  - 1.2
+  - 1.2.11
```

Comment:
`zlib` is [kind of a mess]( https://abi-laboratory.pro/tracker/timeline/zlib/ ). Pinning exactly seems safest.